labgrid/resource/udev.py

```diff
@@ -456,6 +456,40 @@ def __attrs_post_init__(self):
         self.match['DRIVER'] = 'hub'
         super().__attrs_post_init__()
 
+@target_factory.reg_resource
+@attr.s(eq=False)
+class SISPMCTLPowerPort(USBResource):
+    """This resource describes a SiS-PM (Silver Shield PM) Control power port.
+
+    Args:
+        index (int): port index
+    """
+    index = attr.ib(default=None, validator=attr.validators.instance_of(int))
+
+    def filter_match(self, device):
+        match = (device.properties.get('ID_VENDOR_ID'), device.properties.get('ID_MODEL_ID'))
+
+        if match not in [
+                ("04b4", "fd10"),
+                ("04b4", "fd11"),
+                ("04b4", "fd12"),
+                ("04b4", "fd13"),
+                ("04b4", "fd15")
+                ]:
+            return False
+
+        return super().filter_match(device)
+
+    # Overwrite the avail attribute with our internal property
+    @property
+    def avail(self):
+        return bool(self.index)
```

Comment:
This seems wrong. How is this supposed to determine if the port is available?

Reply:
Am I missing something here? This has not been addressed yet, right?

Reply:
No it's not. I'll have a look now. But it might take time to get this fixed by me. I'm not a python developer so bare with me.

Reply:
seems like I don't need that. I'm able to discover the device without overwriting avail function. I'll delete this.